src/mailtemplates/schedule-change-conflict/scheduled-change-conflict.plain.mustache

```diff
@@ -0,0 +1,9 @@
+Conflict detected in a scheduled change
+
+{{ conflict }} Scheduled change requests that use this {{ conflictScope }} can no longer be applied and their scheduled applications will fail{{#conflictResolution}}{{.}}{{/conflictResolution}}{{^conflictResolution}}.{{/conflictResolution}}
```

Comment:
I'm not entirely sure how the syntax works here, but just make sure there is a space before the conflict resolution text.